### DISCUSSION TITLE
How can I update a specific npm package to the latest version?

### DISCUSSION THREAD

**Discussion by HoussemZorgui** [community/community#140308] [npm] [answered]
I have a project using several npm packages, and I want to update a specific package, for example, express, to the latest version. What command should I use to do this? Are there any potential issues I should be aware of during the update?

**Comment by HoussemZorgui** [comment]
How can I update a specific npm package to the latest version?

**Comment by t1noo7** [comment, marked as answer]
npm update [package] will do what you want as long as it is a patch or minor version bump,and it will automatically take you to the latest destination.

**Reply by HoussemZorgui** [reply]
Thank you

**Reply by t1noo7** [reply]
> Thank you
gladly to make you happy. Hopefully you have a happy time with a gf PC, cheer! ^_<
